File: day060/homework/code/codewars10.py
# https://www.codewars.com/kata/517abf86da9663f1d2000003/train/python
def to_camel_case(text):
    if not text:
        return ""
    words = text.replace('-', '_').split('_')
    return words[0] + ''.join(w.capitalize() for w in words[1:])

File: day068/homework/code/codewars10.js
// https://www.codewars.com/kata/5552101f47fc5178b1000050/train/python
def dig_pow(n, p):
    total = sum(int(digit) ** (p + i) for i, digit in enumerate(str(n)))
    return total // n if total % n == 0 else -1
